src/modules/account/types/collections.ts

```diff
@@ -1,4 +1,6 @@
-import { MediaTypes } from '@shared/types';
+import { ApiResponseWrapper, MediaTypes } from '@shared/types';
+
+export type GetCollections = ApiResponseWrapper<Collection>;
```

Comment:
this sounds like it is a type of a function, not an object
maybe GetCollectionsResponse is better?